src/main/scripts/src/app/core/header/header.component.scss

```diff
@@ -69,6 +71,9 @@
 }
 
 .profile {
+  min-width: 45px;
+  min-height: 45px;
```

Comment:
The following code does the same thing in our layout:
```css
flex-shrink: 0;
```
But your solution is valid as well.